MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -581,10 +589,37 @@ - (MUXSDKPlayerData *)getPlayerData {
                                                                     encoding:NSUTF8StringEncoding]];
         }
     }
+
+    // Only report sampling data if the playback is live
+    if (_playbackIsLivestream) {
+        // Sampling Data
+        NSTimeInterval currentDate = [_player.currentItem.currentDate timeIntervalSince1970];
+        playerData.playerProgramTime = [NSNumber numberWithLongLong: (long long)(currentDate * 1000)];
+
+
+        if ([_player.currentItem.seekableTimeRanges count] > 0) {
+            // seekableTimeRanges is ordered, so we only need to look at the last one
+            // Note about seekableTimeRanges: the meaning of the values in seekableTimeRanges appears to change
+            // across OS versions and device types. Sometimes the duration appears to take holdbacks from the HLS
+            // manifest into consideration, and other times not. Use this value with caution.
+            CMTimeRange seekableRange = [_player.currentItem.seekableTimeRanges.lastObject CMTimeRangeValue];
+            CGFloat start = CMTimeGetSeconds(seekableRange.start);
+            CGFloat duration = CMTimeGetSeconds(seekableRange.duration);
+            CGFloat timeOfVideo = CMTimeGetSeconds([[_player currentItem] currentTime]);
+            CGFloat livePosition = start + duration;
+            NSTimeInterval viewStartDate = currentDate - timeOfVideo;
+            NSTimeInterval newestProgramDate = viewStartDate + livePosition;
```

Comment:
I think we should call this `liveEdgeProgramTime` rather than `newestProgramDate` to be consistent with the name of the metric. These are relatively minor nits but since the definitions of these values is tricky, feels worth being extra pedantic about the naming :)

Reply:
yeah, i think in general there's a lot of confusion with the way we've chosen to name things. I think we use "time" to refer to dates or a number of seconds into a video. I was trying to avoid that confusion here by using "date" in variable names that represent dates, and "time" to represent numbers of seconds. I can change it to match the metric name but i'd rather change the metric name hahah

Reply:
maybe i'll rename these "timestamp" instead of "date", i think that's a happy medium